src/AutomatedCar/SystemComponents/DummySensor.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    using AutomatedCar.Models;
+    using AutomatedCar.SystemComponents.Packets;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class DummySensor : SystemComponent
+    {
+        DummyPacket dummyPacket = new DummyPacket();
+        int circleX;
+        int circleY;
+        public DummySensor(VirtualFunctionBus virtualFunctionBus) : base(virtualFunctionBus)
+        {
+            virtualFunctionBus.DummyPacket = this.dummyPacket;
+            virtualFunctionBus.RegisterComponent(this);
```

Comment:
Hello I just commented my concern with this line here as well, please checkout:
https://github.com/SzFMV2023-Tavasz/warmup/pull/8#discussion_r1135331935